### ISSUE TITLE
ForDevelopers.md guide

### ISSUE THREAD

**Issue by Stivius** [Stivius/xibo-linux#206]
Write a guide for new developers who wants to contribute to repository

**Comment by dasgarner** [comment]
Please follow format for our other open source repositories: https://github.com/xibosignage/xibo-cms#contributing

**Comment by Stivius** [comment]
@dasgarner this guide should be for those who want to contribute but don't know anything about the codebase. I mean code structure, code style, project architecture. It's just a note with recommendations. The more developers come the more difficult it will be to maintain the codebase without any guides.

**Comment by dasgarner** [comment]
Sure, but first we need to mention that for any contributions to be
accepted the developer must be willing to sign the CLA.

The other things you speak of can come after that I think?

-

**Comment by Stivius** [comment]
@dasgarner, yes, you are right. All these recommendations should be applied after signing CLA.

**Comment by Stivius** [comment]
Relates to https://github.com/xibosignage/xibo-linux/issues/117 and https://github.com/xibosignage/xibo-linux/issues/116
